src/static/lookup.php

```diff
@@ -77,4 +78,33 @@
 
 </body> 
 
-</html>
\ No newline at end of file
+</html>
```

Comment:
this page needs to be rearranged. it's a bit weird having the php in 2 different locations. not intuitive to read and understand.